bin/kubectl-crossplane-stack-list

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+function usage {
+  # The reason for putting stack name before stack image source is it seems like stack name
```

Comment:
is this comment a copy/paste relic from another command? i don't see stack name or stack image source declared in this command.